packages/video_player/lib/video_player.dart

```diff
@@ -584,16 +706,38 @@ class _VideoScrubberState extends State<_VideoScrubber> {
 /// [padding] allows to specify some extra padding around the progress indicator
 /// that will also detect the gestures.
 class VideoProgressIndicator extends StatefulWidget {
+  /// Construct an instance that displays the play/buffering status of the video
+  /// controlled by [controller].
+  ///
+  /// Defaults will be used for everything except [controller] if they're not
+  /// provided. [allowScrubbing] defaults to false, and [padding] will default
```

Comment:
nits: `The [allowingScrubing]...`
I think we don't start sentences with small letter characters

Reply:
It's weird to start off a sentence with "[a", but I think this is actually correct here for a few reasons.

Using  "the" would imply that `allowScrubbing` is significant somehow--that it's a specific `allowScrubbing` out of multiple possible `allowScrubbing`s out there, or maybe somehow very important. Similarly if I were to use "a/an" I would be implying a random `allowScrubbing` out of the various `allowScrubbing`s available. But In this case there is only one parameter called `allowScrubbing`, so it doesn't make sense to use either article. I'm referring to the one and only `allowScrubbing` throughout, and it's just a regular parameter like any other one.

And then after that normally sentences do start with upper case letters. This is where it gets into extremely confusing territory because prescriptive grammarians haven't really caught on to coding documentation yet, but here I'm basically using `[allowScrubbing]` as a name that is also lower cased. That's unusual for English, but when it does happen it is correct to just use the proper name as it's normally capitalized instead of upper casing it, the same if I were talking about iPhones. See https://english.stackexchange.com/a/68665 for some discussion on it.

That said, English doesn't really have real grammar rules so this is all very subjective.